lib/smart_answer/calculators/benefit_cap_calculator_data_query.rb

```diff
@@ -1,12 +1,15 @@
 module SmartAnswer::Calculators
   class BenefitCapCalculatorDataQuery
-    attr_reader :data, :benefits, :questions
     def data
       @data ||= YAML.load_file(Rails.root.join('lib', 'data', 'benefit_cap_data.yml'))
     end
 
+    def rates
```

Comment:
As I've mentioned elsewhere, I think this would be better named something like `weekly_benefit_cap`.